src/util.ts

```diff
@@ -69,29 +71,21 @@ export function convertStringtoSymbolKind(kind: string): SymbolKind {
 }
 
 export function path2uri(root: string, file: string): string {
-	let ret = 'file://';
-	if (!strict && process.platform === 'win32') {
-		ret += '/';
-	}
 	let p;
 	if (root) {
 		p = resolve(root, file);
 	} else {
 		p = file;
 	}
-	p = toUnixPath(p).split('/').map(encodeURIComponent).join('/');
-	return ret + p;
+	return Uri.file(p).toString();
```

Comment:
VS Code's URI module is broken in a couple ways. For example, it encodes `file://c:/whatever` as `file://c%3a/whatever`, while node's `url.format()` behaves correctly. I would prefer not to use it as a dependency. What is the difference in output here?